src/Storage/Adapter/Cookie.php

```diff
@@ -66,76 +65,64 @@ public function __construct($cookieName, $ttl = 157766400)
      */
     protected function parseExistingCookie()
     {
-        if (is_array($this->cookieValues)) {
+        if (is_array($this->data)) {
             return;
         }
 
         $cookiesContent = filter_input_array(INPUT_COOKIE);
 
         if (empty($cookiesContent) || !isset($cookiesContent[$this->cookieName])) {
-            $this->cookieValues = [];
+            $this->data = [];
             return;
         }
 
         $deserializedCookie = json_decode($cookiesContent[$this->cookieName], true);
 
         if (is_null($deserializedCookie)) {
-            $this->cookieValues = [];
+            $this->data = [];
             return;
         }
 
-        $this->cookieValues = $deserializedCookie;
+        $this->data = $deserializedCookie;
     }
 
     /**
-     * Saves cookie with serialized test values
-     *
-     * @return bool
+     * {@inheritDoc}
      */
     protected function saveCookie()
     {
         $this->parseExistingCookie();
 
-        return setcookie($this->cookieName, json_encode($this->cookieValues), time() + $this->ttl, '/');
+        return setcookie($this->cookieName, json_encode($this->data), time() + $this->ttl, '/');
     }
 
     /**
      * {@inheritDoc}
-     *
-     * @param string $identifier The tests identifier
      */
     public function has($identifier)
     {
         Assert::string($identifier, 'Test identifier is invalid.');
         Assert::notEmpty($identifier, 'Test identifier is invalid.');
-        
+
         $this->parseExistingCookie();
 
-        return isset($this->cookieValues[$identifier]);
+        return isset($this->data[$identifier]);
```

Comment:
I think we should use `array_key_exists`, isset fails if the value is '0' for example.